contrib/docker/README.md

```diff
@@ -31,9 +10,11 @@ Specify a ``SYNAPSE_CONFIG_PATH``, preferably to a persistent path,
 to use manual configuration. To generate a fresh ``homeserver.yaml``, simply run:
 
 ```
-docker-compose run --rm -e SYNAPSE_SERVER_NAME=my.matrix.host synapse generate
+docker-compose run --rm -e SYNAPSE_SERVER_NAME=my.matrix.host -e SYNAPSE_REPORT_STATS=yes synapse generate
 ```
 
+Above command contains variable SYNAPSE_REPORT_STATS which is opt-in only. You are free to say 'no' here. 
```

Comment:
Currently this reads:

> You are free to say 'no' here. This will also generate necessary signing keys. 

... which is misleading. I'm suggesting you reword this by adding some more text above the example.

```suggestion
```

Reply:
I have made some changes here.

Reply:
Still not sure where the stats are sent.  Is this simply telling the client to send some status to their home synapse server?